src/spikeinterface/preprocessing/unsigned_to_signed.py

```diff
@@ -15,17 +15,18 @@ class UnsignedToSignedRecording(BasePreprocessor):
     def __init__(
         self,
         recording,
+        bit_depth=None,
```

Comment:
If None, it should automatically use the full range IMO:

e.g., `int16` --> `bit_dept=16`

What do you think @BptGrm ?